config/projects/fuzzing.yml

```
@@ -683,3 +717,12 @@ fuzzing:
         - secrets:get:project/fuzzing/deploy-npm
       to:
         - hook-id:project-fuzzing/gr-idl-update
+    - grant:
+        - docker-worker:capability:privileged
+        - queue:create-task:highest:proj-fuzzing/ci
+        - queue:get-artifact:project/fuzzing/prefmonitor/*
+        - queue:route:notify.email.jkratzer@mozilla.com.on-failed
+        - queue:scheduler-id:-
+        - secrets:get:project/fuzzing/prefmonitor-ci-token
+        to:
+          - hook-id:project-fuzzing/prefmonitor
```

You've got a small yaml lint issue. Just need to tab lines 727, 728 back one.

```suggestion
      to:
        - hook-id:project-fuzzing/prefmonitor
```